extension/src/popup/helpers/soroban.ts

```diff
@@ -102,24 +103,31 @@ export const getTokenBalance = (
   );
 };
 
-export const contractIdAttrToHex = (byteArray: Buffer) =>
-  byteArray.reduce(
-    (prev, curr) =>
-      // eslint-disable-next-line
-      prev + ("0" + (curr & 0xff).toString(16)).slice(-2),
-    "",
-  );
```

Comment:
we actually can simply call `toString` now rather than having to do something complicated like this 🎉